app/serializers/v2/move_serializer.rb

```diff
@@ -22,16 +22,14 @@ class MoveSerializer
                :time_due,
                :updated_at
 
-    has_one :profile, serializer: V2::ProfileSerializer
-    has_one :from_location, serializer: LocationSerializer
-    has_one :to_location, serializer: LocationSerializer
+    has_one :from_location,          serializer: LocationSerializer
     has_one :prison_transfer_reason, serializer: PrisonTransferReasonSerializer
-    has_one :supplier, serializer: SupplierSerializer
+    has_one :profile,                serializer: V2::ProfileSerializer
+    has_one :supplier,               serializer: SupplierSerializer
+    has_one :to_location,            serializer: LocationSerializer
 
     has_many :court_hearings, serializer: CourtHearingSerializer
-    has_many :events, serializer: GenericEventSerializer do |object|
-      object.generic_events.applied_order
-    end
+    has_many :timeline_events, type: 'events', serializer: GenericEventSerializer, &:all_events_for_timeline
```

Comment:
Does this still work without specifying `type` here? The serializer should handle that I think 🤔